jakarta.batch.tck.api/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I wanted to have a separate module for the code that isn't specific to Batch TCK. In the future it could be easily copied to other specs that follow our approach or even extracted to a separate EE4J project. But I renamed the module to tck.util in the new PR https://github.com/eclipse-ee4j/batch-tck/pull/40 because it's really more a utility module with functional code rather than abstract API.